packages/core/src/lib/session.ts

```diff
@@ -14,20 +15,31 @@ export const assignInteractionResults = async (
   // have to do it manually
   // refer to: https://github.com/panva/node-oidc-provider/blob/c243bf6b6663c41ff3e75c09b95fb978eba87381/lib/actions/authorization/interactions.js#L106
   const details = merge ? await provider.interactionDetails(ctx.req, ctx.res) : undefined;
+  const ts = dayjs().unix();
+  const mergedResult = {
+    // Merge with current result
+    ...details?.result,
+    ...result,
+  };
 
   const redirectTo = await provider.interactionResult(
     ctx.req,
     ctx.res,
     {
-      // Merge with current result
-      ...details?.result,
-      ...result,
+      ...mergedResult,
+      login: mergedResult.login
+        ? {
+            ...mergedResult.login,
+            // Update ts(timestamp) if the accountId is been set in result
+            ts: result.login?.accountId ? ts : mergedResult.login.ts,
```

Comment:
nit: Can we just call it `timeStamp` instead of `ts`?

Reply:
Also why is `ts` only updated when accountId is set? Why don't we always update `ts` regardless of `accountId`?

Reply:
You can find `ts` in https://github.com/panva/node-oidc-provider/blob/main/docs/README.md

<img width="653" alt="image" src="https://user-images.githubusercontent.com/5717882/189867853-34ead5e3-cfb2-4132-a2c7-8d214081ecbe.png">

Reply:
OK, no problem then